third_party/blink/renderer/core/brave_page_graph/graph_item/edge/event_listener/edge_event_listener_action.cc

```diff
@@ -35,10 +36,12 @@ ScriptId EdgeEventListenerAction::GetListenerScriptId() const {
 }
 
 ItemDesc EdgeEventListenerAction::GetItemDesc() const {
-  return GraphEdge::GetItemDesc() + " [" + event_type_ + "]" +
-         " [listener id: " + base::NumberToString(listener_id_) + "]" +
-         " [listener script id: " +
-         base::NumberToString(GetListenerScriptId()) + "]";
+  WTF::TextStream ts;
+  ts << GraphEdge::GetItemDesc() << " [" << event_type_ << "]"
+     << " [listener id: " << base::NumberToString(listener_id_) << "]"
+     << " [listener script id: " << base::NumberToString(GetListenerScriptId())
```

Comment:
one more thing: `base::NumberToString` should be removed as they should be natively supported by `WTF::TextStream`.

Reply:
Done. For this and all the other occurrences.

Reply:
what about `base/strings/string_number_conversions.h` includes?

Reply:
Done